src/commands/upgrade.js

```diff
@@ -16,7 +14,7 @@ const { stop, start, upgradeEnv } = require( '../environment' );
 exports.command = 'upgrade';
 exports.desc = false; // @todo: "false" means that this command is hidden
 
-exports.handler = makeCommand( chalk, logSymbols, async ( { verbose } ) => {
+exports.handler = makeCommand( {}, {}, async ( { verbose } ) => {
```

Comment:
If I remember correctly we don't need docker for this command, so we can use `checkDocker: false` for it:

```js
makeCommand( { checkDocker: false }, async ( { verbose } ) => { ... } );
```